crates/bevy_pbr/src/meshlet/from_mesh.rs

```diff
@@ -294,6 +289,7 @@ fn simplify_meshlet_groups(
     let target_error = target_error_relative * mesh_scale;
 
     // Simplify the group to ~50% triangle count
+    // TODO: Simplify using vertex attributes
```

Comment:
Is this about the bevy mesh api or more specific to meshlets?

Reply:
Specific to meshlets. Simplification (i.e. the LOD building) only accounts for position atm, and not things like UVs/normals. See https://github.com/zeux/meshoptimizer/issues/158.